### PR TITLE
Added unit test for getOutClusterCS method

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

**What this PR does / why we need it**:
This adds a unit test for getOutClusterCS in util_test.go file

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes https://github.com/openebs/openebs/issues/799

**Special notes for your reviewer**:

### CODE REVIEW

File: orchprovider/k8s/v1/util_test.go [hunk 1]
```diff
@@ -137,3 +137,26 @@ func TestK8sUtilServices(t *testing.T) {
...
 	}
 }
+
+// TestGetOutClusterCS tests the working of TestgetOutClusterCS() method.
+func TestGetOutClusterCS(t *testing.T) {
+	// a noop pvc that in turn signals use of defaults
+	pvc := &v1.Volume{
+		Namespace: "default",
+	}
+	//pvc.Labels = map[string]string{}
```
Comment (kmova) [inline orchprovider/k8s/v1/util_test.go]: remove this commented code.

**Comment by kmova** [comment]
@ashishranjan738  Thanks for raising this PR.

Can you improve the test case, so that it follows the pattern explained in this wiki?https://github.com/golang/go/wiki/TableDrivenTests

Use the table approach, we will need to cover both positive and negative code paths.

File: orchprovider/k8s/v1/util_test.go (hunk 1 above)
Comment (ashishranjan738) [reply orchprovider/k8s/v1/util_test.go]: ok sir

Comment (ashishranjan738) [reply orchprovider/k8s/v1/util_test.go]: should i raise a pr once again ?

Comment (prateekpandey14) [reply orchprovider/k8s/v1/util_test.go]: you can commit the new changes here again.

**Comment by prateekpandey14** [comment]
@ashishranjan738 please rebase with master to resolve the conflicts.

**Comment by kmova** [comment]
The issue referenced and the code changes don't match. Please raise a new PR with correct information.

**Pull request closed without merging**